src/layouts/sidebar-sidecar/sidebar-sidecar-layout.module.css

```diff
@@ -50,47 +86,82 @@ for further details.
   }
 }
 
+.mainAreaAndFooter {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+/**
+ * Main area row
+ */
+
 .mainArea {
   display: flex;
   flex-grow: 1;
   min-width: 0;
   justify-content: center;
   padding-bottom: 32px;
   padding-top: var(--main-area-top-padding);
-  padding-left: var(--main-area-padding-left-right);
-  padding-right: var(--main-area-padding-left-right);
+  padding-left: var(--main-area-padding-left);
+  padding-right: var(--main-area-padding-right);
 }
 
 .main {
   align-self: flex-start;
   flex-grow: 1;
   min-width: 0;
-  max-width: calc(780px + (var(--main-area-padding-left-right) * 2));
-  padding-left: var(--main-area-padding-left-right);
-  padding-right: var(--main-area-padding-left-right);
```

Comment:
I've removed duplicate `padding` settings. The intent here is to make the left and right padding of the main content area easier to reason about; but it did require a few changes in related files.